src/Api/MarketingCloud/Messaging.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Oowlish\Salesforce\Api\MarketingCloud;
+
+use Oowlish\Salesforce\Api\Resource;
+
+class Messaging extends Resource
+{
+    /**
+     * Sends transactional email using Marketing Cloud's triggered send functionality.
+     * In order to use this service, configure a triggered send definition in Email Studio.
+     *
+     * @param string $triggeredSendDefinitionId
+     * @param array  $from
+     * @param array  $to
+     * @param array  $options
+     *
+     * @return array
+     */
+    public function sendEmail(string $triggeredSendDefinitionId, array $from, array $to, array $options): array
```

Comment:
```suggestion
    public function sendEmail(string $triggeredSendDefinitionId, array $from, array $to, array $options = []): array
```

Do we really need the `$options` being mandatory?

Reply:
Nope. Good catch. I'll fix it.